shop/src/app/components/presentational/products-details-view/products-details-view.component.html

```diff
@@ -0,0 +1,23 @@
+<div *ngIf="product">
+    <div class="top">
```

Comment:
This `"top"` class name is not quite specific, it would be better if the names were more on point. Something like "os_top-alignment" or anything related, in which "os" stands for "name": "online-shop" (project name acronym) from package.json

Reply:
Done